### ISSUE TITLE
Accessibility concern in dashboard component

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/chapter#1975]
### Discuss your ideas or share your views: 
In dashboard component we have `actions` that act as link (interactive component), here is an example

https://github.com/freeCodeCamp/chapter/blob/7a21f751446a1a4c715edb4c7f9824a9744e3bd3/client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx#L14-L36?plain=1

```ts
const actionLinks = [
  {
    colorScheme: 'blue',
    size: 'xs',
    href: (id: number) => `/dashboard/chapters/${id}/new-venue`,
    text: 'Add venue',
    requiredPermission: Permission.VenueCreate,
  },
  {
    colorScheme: 'blue',
    size: 'xs',
    href: (id: number) => `/dashboard/chapters/${id}/new-event`,
    text: 'Add Event',
    requiredPermission: Permission.EventCreate,
  },
  {
    colorScheme: 'blue',
    size: 'xs',
    href: (id: number) => `/dashboard/chapters/${id}/edit`,
    text: 'Edit',
    requiredPermission: Permission.EventEdit,
  },
];
```

Although this is fine, I am worried about the repetition of the actions, which lead to this accessibility concern.

So here is what I am worried about, we have multiple `edit` button or alike, that do the same thing `edit chapter` but for different chapters, so people may get confused of the lack of info in these buttons and have a worse experience.

I am wondering if we should add `aira-label` to these buttons https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label, so people with accessibility know that they are editing a specific chapter, and to give access to voice users playbook https://blog.google/technology/next-billion-users/voice-users-playbook/, because they need to ask the device to interact with a component, and giving a unique name should help them.

**Comment by ojeytonwilliams** [comment]
It seems to me that repeating which chapter you're in three times would be more frustrating than helpful.

Instead, why don't we make it clearer that you're in the action row for a given chapter? i.e. when a user tabs to that row it could announce "Actions for ${chapter.name}" and then should be clear that Add venue, Add event and Edit all refer to that.

**Comment by Sboonny** [comment]
Ya, I like it. adding `tabindex=0` and `aria-label` should do the trick. 👍 Good call.

**Comment by Sboonny** [comment]
wait what about people who use voice playback, using this won't account for them

**Comment by ojeytonwilliams** [comment]
If the screenreader text was ``chapter ${chapter.name}`` wouldn't that do it?

**Comment by Sboonny** [comment]
true, but people who use voice playback ask the browser to press `edit`, but because there are multiple `edits`, the browser will throw its hands in the air and say "nope, can't handle the request". that's why we need to add unique `aria-label` or `sr-only` for people who use their voice to navigate in the browser.

**Comment by ojeytonwilliams** [comment]
Oh, I didn't know, thanks for explaining.

In that case your original idea of using `aria-label` seems like the way to go.

**Comment by bbsmooth** [comment]
@Sboonny You are correct, these interactive elements should have a unique accessible name. For the action buttons on the dashboard page my suggestion would be to just add the name of the chapter to the button text using sr-only text. For example, if the chapter name is "Skiles and Sons" then the "Add venue" button text would be `Add venue <span class='sr-only'>Skiles and Sons</span>`. 

Also, the "Add new" button in the top right should have sr-only "chapter" added to it as well.

If you want to do this with `aria-label` instead that's fine. I personally try to use sr-only text whenever possible because if people are using translation services on the page then often the text in `aria-label` will not get translated.

**Comment by Sboonny** [comment]
Thank you so much, this is an awesome holiday gift. I was worried about not being able to update `aria-label` in the `DataTable`, and thought we weren't going to provide the best experiences we could, it's good to know that we are traversing the right path. I will update "Add new" buttons 👍

**Comment by Sboonny** [comment]
This PR address the link, so all should be good now https://github.com/freeCodeCamp/chapter/pull/2067
